Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -288,7 +288,7 @@ class RenderStyle {
 
     static RenderStyle& defaultStyle();
 
-    static RenderStyle create();
+    WEBCORE_EXPORT static RenderStyle create();
```

Comment:
*disclaimer: I'm not an official reviewer*.

I'm not sure if we should to do that. You can probably remove your .cpp test since you have introduced a web test.

Reply:
I think this is okay -- it's being exposed for a TestWebKitAPI purpose. We don't really have a good mechanism to expose things only for API tests/Unit Tests, which would be a good project for someone.